pkg/apis/etcd/v1beta1/register.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package spec
+package v1beta1
```

Comment:
This package is `v1beta1` but the `SchemeGroupVersion` on L33 is `v1beta2`. Should these match?